### ISSUE TITLE
Release Stylish as official Safari Extension

### ISSUE THREAD

**Issue by nikivdev** [350d/stylish#55]
I would love to help you build this. Either we can donate money to @350d to pay the license fee or keep the extension paid. The extension is very valuable to have.

Do you have any plans on this? @350d

**Comment by 350d** [comment]
Hello! Latest and updated version is under review for more than a month already and all I have is - "please wait" from Apple.

**Comment by mloureiro** [comment]
is there any update on this topic?

**Comment by axeII** [comment]
I found this cool project **Dark Reader** for those who want dark mode everywhere and supports safari now. https://github.com/darkreader/darkreader

**Comment by jspiro** [comment]
@L3rchal while I appreciate your link, this is not the purpose of stylish and is off-topic.

**Comment by wsy** [comment]
Hi,
Is there any update on this?
Is this Safari Extension still under review?
The old download page still says "The Safari Extension you've requested is not currently available".

Your extension is very useful!
Will you consider making an application and release it on Mac App Store?

**Comment by riconaranjo** [comment]
Reluctantly here is a [link](
https://itunes.apple.com/ca/app/cascadea/id1432182561?mt=12) to Cascadea, a Mac store app that supports userstyles.org styles just like stylish / stylus.

I do appreciate the work @350d put into this extension and I'm sad that this issue hasn't been resolved yet.

**Comment by grogcw** [comment]
Any news by Apple....?

I just crashed my old install and the extension doesn't work anymore (sad).

**Comment by Rhywun** [comment]
Does anyone know where Stylish for Safari stores your styles? The plug-in doesn't work with macOS Catalina and if I want to recreate my styles I need to locate them first.

**Comment by ws909** [comment]
This is a legacy extension. The last version of Safari to support them, is Safari 12. (macOS 10.14 Mojave). The new Swift API doesn't support injection of scripts into webpages other than those bundled with the app, set in the .plist file.

**Comment by Rhywun** [comment]
@ws909 - Understood. I'm trying to find the config files from my formerly working Stylish plug-in so I can copy my styles and maybe use them with some other solution that might come up - or with another browser if I go that route.  Thanks

**Comment by ws909** [comment]
@Rhywun Yeah, it's quite annoying the new API lacks so much. An alternative, very **cumbersome** solution, is to make your own personal extension, and just update and reinstall it manually each time you make a change to a CSS file, or add a new one.
